app/controllers/users.js

```diff
@@ -1,38 +1,89 @@
-import jwt from 'jsonwebtoken';
-import nodemailer from 'nodemailer';
-import mongoose from 'mongoose';
+/**
+ * Module dependencies.
```

Comment:
Please revert this file back to how it was on staging and add your changes afterward. When fixing conflicts, ALWAYS accept the incoming changes from staging and add your changes afterward, not the other way round.